evap/evaluation/models.py

```diff
@@ -934,6 +935,11 @@ class Meta:
         verbose_name = _("question")
         verbose_name_plural = _("questions")
 
+    def save(self, *args, **kwargs):
+        if self.type in [Question.TEXT, Question.HEADING]:
+            self.allows_textanswers = False
```

Comment:
but... isn't that weird? a text question doesn't allow textanswers?

It's a bit verbose but I'd suggest naming this `allows_additional_textanswers`.

Reply:
done